docs/README.md

```diff
@@ -50,6 +50,7 @@ The `condition-layout.product` block provides the subjects below to use in `cond
 | `selectedItemId`  | `value` | Id of the current selected SKU |
 | `productClusters` | `array` | List of product clusters.             |
 | `categoryTree`    | `array` | List of categories.                   |
+| `properties`      | `array` | List of product-specifications.       |
```

Comment:
I think `specifications` would be a better fit here. What do you think?

Reply:
Or `specificationProperties`

Reply:
Hello @kaisermann and @victorhmp!

If possible, I wanted your help on two points.

First, I am having problems with the App IO test, and in my development branch the test is OK.

And second, I'm having trouble changing the name of the prop from "properties" to "specificationProducts", as you suggested. Could you help me how can I do that, please?

Best Regards!

Reply:
Hey @LucasCastroAcctGlobal 👋  The tests seem to be passing now 🤷. About the prop name, you could rename it when destructuring the `product` context value and then passing the value, with the new name, to the `useMemo`:

```js
 const {
    productId,
    categoryId,
    brandId,
    productClusters,
    categoryTree,
    properties: specificationProperties,
  } = product ?? {}
  const { itemId: selectedItemId } = selectedItem ?? {}

  // We use a useMemo to modify the a condition layout "values"
  // only when some of the context props change.
  const values = useMemo(
    () => ({
      productId,
      categoryId,
      brandId,
      productClusters,
      categoryTree,
      selectedItemId,
      specificationProperties,
    }),
    [
      brandId,
      categoryId,
      categoryTree,
      productClusters,
      productId,
      selectedItemId,
      specificationProperties,
    ]
  )```

Reply:
Hello @kaisermann!

Thanks a lot for the help! I made the modifications and tests and it worked.

Let me know if there is anything else to fix!